web/src/components/shared/Ads/Ad.tsx

```diff
@@ -60,16 +63,36 @@ const WarningBlock: FC<{ unactiveReason: string }> = ({ unactiveReason }) => (
   </Box>
 );
 
+const calculateRateDiff = (advertRate: Money, currentRate: Money): number => {
+  const diff = Number(
+    parseNumeric(advertRate.amount.div(currentRate.amount).times(100).sub(100).toString(), {
+      maxFractionDigits: 2,
+      allowNegativeNumeric: false,
+    }),
+  );
+
+  return diff;
+};
```

Comment:
```suggestion
const calculatePercentageDiff = (prevRate: Money, nextRate: Money): number => nextRate.amount.div(prevRate.amount).sub(1).times(100).toNumber();
```
Так будет лучше, над названием можно ещё подумать :)

Reply:
чет я не доглядел toNumber() в пакете

Reply:
может просто percentageDiff?

Reply:
У функции должно быть действие в названии https://github.com/kettanaito/naming-cheatsheet#ahclc-pattern

Reply:
не нужно думать над названием функции, если ее нет :)

Reply:
а гайдик хороший